clinica/pipelines/dwi_preprocessing_using_t1/dwi_preprocessing_using_t1_workflows.py

```diff
@@ -91,6 +95,17 @@ def epi_pipeline(name="susceptibility_distortion_correction_using_t1"):
     function (SyN algorithm). This workflow allows also a coregistration of DWIs with their respective
     baseline T1-weighted structural scans in order to latter combine tracks and cortex parcellation.
 
+    Parameters
+    ----------
+    base_dir: str
+    Working directory, whcih contains all of the intermediary data generated.
+
+    delete_cache: bool
+    If True, part of the temporary data is automatically deleted after usage.
```

Comment:
```suggestion
        If True, part of the temporary data is automatically deleted after usage.
```

Could be worth documenting what data will be deleted by using this option. WDYT?